app/models/account.rb

```diff
@@ -138,6 +138,10 @@ def notify_users
     [owner_user] + business_admin_users
   end
 
+  def send_notification(statement)
```

Comment:
Another thing where I don't necessarily feel strongly on, but this is seems like it's part of the responsibility of the StatementCreator.